app/controllers/v0/id_card_attributes_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require 'vic_helper'
+
+module V0
+  class IdCardAttributesController < ApplicationController
+    before_action :authorize
+
+    def show
+      id_attributes = IdCardAttributes.for_user(current_user)
+      vic_url = VIC::Helper.generate_url(id_attributes)
+      render json: { 'redirect' => vic_url }
+    rescue => e
+      # TODO: Tighten this up. Only likely application error is branch of service data from eMIS.
+      # Could also get a more general error in signing request.
+      log_exception_to_sentry(e)
+      raise Common::Exceptions::Forbidden, detail: 'Could not verify military service attributes', code: 'VIC005'
+    end
+
+    private
+
+    def authorize
+      raise Common::Exceptions::Forbidden, code: 'VIC001' unless current_user.loa3?
+      # TODO: possible change to more specific exceptions with actionable codes
+      raise Common::Exceptions::Forbidden, detail: 'Unable to verify EDIPI', code: 'VIC002' unless
+        current_user.edipi.present?
+      begin
+        raise Common::Exceptions::Forbidden, detail: 'Not eligible for a Veteran ID Card', code: 'VIC003' unless
+          current_user.veteran?
+      rescue => e
```

Comment:
So I know you have to do this in case eMIS is down, but it might be confusing for an authorize block. Maybe some comments would be helpful here. Also, instead of modifying Common::Exceptions::Forbidden, why not use BackendServiceException and i18n?

Reply:
Yeah...I dislike the `veteran?` interface and the need to catch esoteric exceptions from it...I'll think about how to refine this.